tls/s2n_psk.c

```diff
@@ -220,6 +220,7 @@ struct s2n_offered_psk* s2n_offered_psk_new()
 {
     DEFER_CLEANUP(struct s2n_blob mem = { 0 }, s2n_free);
     GUARD_PTR(s2n_alloc(&mem, sizeof(struct s2n_offered_psk)));
+    GUARD_PTR(s2n_blob_zero(&mem));
 
     struct s2n_offered_psk *psk = (struct s2n_offered_psk*)(void*) mem.data;
     *psk = (struct s2n_offered_psk){ 0 };
```

Comment:
You don't need to do "*psk = (struct s2n_offered_psk){ 0 };" anymore. You've already zeroed the memory.